Description: test/avl: Reduce test AVL tree size.
 So that the testsuite doesn't take forever on slower machines.
 .
 Reported-by: Christian T. Steigies <cts@debian.org>
 Signed-off-by: Lennert Buytenhek <buytenh@wantstofly.org>
Author: Lennert Buytenhek <buytenh@wantstofly.org>
Date:   Mon Jul 29 16:56:54 2013 +0200

--- ivykis-0.36.2.orig/test/avl.c
+++ ivykis-0.36.2/test/avl.c
@@ -133,7 +133,7 @@ static void tree_check(struct iv_avl_tre
 }
 
 
-#define NUM	16384
+#define NUM	1024
 
 static struct node *f[NUM];
 
@@ -165,7 +165,7 @@ int main()
 {
 	int i;
 
-	alarm(300);
+	alarm(180);
 
 	srand(time(NULL) ^ getpid());
 
